test/helpers/getPerformanceTests.ts

```diff
@@ -0,0 +1,100 @@
+import testFs from './testFs';
+import {
+	evaluateXPathToNodes,
+	evaluateXPathToString,
+	evaluateXPathToFirstNode,
+} from '../../src/index';
+
+import {
+	ALL_TESTS_QUERY,
+	getAllTestSets,
+	getFile,
+	unrunnableTestCasesByName,
+} from './qt3TestsTools';
+
+const PERFORMANCE_TESTS_FILE = 'runnablePerformanceTestNames.csv';
+const TOTAL_NUMBER = 6420;
+let numberOfTest = 50;
+
+const createNewRunnableTests = () => {
+	const runningPossibilityForEachTest = TOTAL_NUMBER / numberOfTest;
+	const allTestSets = getAllTestSets();
+	const newRunnableTests = new Set();
+
+	function addTestNames() {
+		for (const testSetFileName of allTestSets) {
```

Comment:
so this basically takes 50 tests of each set, or 50 tests overall?

Reply:
I need `regenerateIndex` in the line 90 for `newNumber`.
This takes 50 tests in total for all qt3tests.